test/executables/react16_ci_precheck_test.dart

```diff
@@ -160,23 +197,13 @@ ProcessResult runCiPrecheck({String onDirectory}) {
   return result;
 }
 
-ProcessResult runCiPrecheckWithFakeDartProject(
-    {PubspecCreator pubspecCreator, String mainDartContents}) {
-  var testPackage = DartTempProjectCreator(
-      pubspecCreator: pubspecCreator ?? tansitionPubspecCreator,
```

Comment:
`pubspecCreator` was always specified, meaning this default could be safely removed